frontend/src/dashboard/panels/EventInvites.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import useAxios from 'axios-hooks';
+import {
+  Box,
+  CircularProgress,
+  Paper,
+  Typography
+} from '@material-ui/core';
+
+import { EventInvite } from '../../types/interfaces';
+
+// NOTES: No information about event in Invite
+
+const mockInvites: EventInvite[] = [
+  {
+    event_id: 1, 
+    event_invite_id: 1,  
+    accepted: null,
+    recipient_id: 2,
+    sender_id: 1,
+    invite_time: new Date()
+  }
+]
+
+const EventInvites = () => {
+    const [{ data, loading, error }] = useAxios<[EventInvite]>({
+      url: `/api/invites`,
+    }, { manual: false, useCache: false });
+
+    return (
+      loading 
+        ? <CircularProgress color="secondary"/>
+        : (
+            !error
+              ? <Typography variant="body1">There was an error: {JSON.stringify(error)}</Typography>
+              : <>
+                  {mockInvites.map((invite) => (
+                    <Paper elevation={2} key={invite.event_invite_id}>
+                      <Box p={1}>
+                        <Typography variant="body1">Invite ID: {invite.event_invite_id}</Typography>
+                        <Typography variant="body1">Sender ID: {invite.sender_id}</Typography>
+                        <Typography variant="body1">Event ID: {invite.event_id}</Typography>  
```

Comment:
We haven't really discussed this much, I think it would be useful to display a pair of accept and decline buttons straight from the invite notif on the dashboard. TBH it's probably one of the features we won't be able to implement functionality for in time, but I think it would pretty easy to just put two greyed out buttons? In any case, I think it's more useful than just listing a bunch of ID's being listed out. Maybe once the database is connected properly, some querying can happen to get the proper string data.

Reply:
Yeah this definitely wont be the final data shown. It will hopefully be more like the mockups once the API is in place. I think I had an accept button in the mockup but I could add a decline button, too.